Snowdog_Components/components/Organisms/more-info/_more-info.scss

```diff
@@ -2,6 +2,7 @@
 
 .more-info {
     pointer-events: all;
+    margin: 0 0 $spacer;
```

Comment:
add variable for it, the same as in previous commit but with different name
`$more-info__margin`